pkg/konfluxgen/konfluxgen.go

```diff
@@ -164,6 +171,20 @@ func Generate(cfg Config) error {
 
 			buf.Reset()
 
+			imageRepositoryPath := filepath.Join(cfg.ResourcesOutputPath, "applications", appKey, "components", "imagerepositories", fmt.Sprintf("%s.yaml", componentKey))
```

Comment:
maybe we could use the same directory and flatten the structure in the same `components` directory so that navigating is easier and in file explorers we would get 

- <component>.yaml
- <image-repository-for-component>.yaml

Reply:
I am fine with both. I just thought about having it in a separate dir to not mix it too much, as we have already (depending on the number of components) some files in the dir and the imagerepository ones are probably not so relevant for the user :shrug: 
But as said, I am fine with both...